DOC.md

```diff
@@ -223,9 +223,9 @@ locations.
 
 #### Set of locations
 
-The set of locations is expected as an `{fset Location }` using the finite
+The set of locations is expected as a `{fset Location }` using the finite
 sets of the [extructures] library. For our purposes, it is advisable to write
-them directly as list which of locations which is then cast to an `fset` using
+them directly as list which of locations which is then cast to a `fset` using
```

Comment:
Why add typos? "an `fset`" is the correct spelling.
Also it's mostly unrelated to the problem at hand.